### PR TITLE
bpo-39009: Fix typo in test__locale

### PR DESCRIPTION
**Pull request by timgates42**
There is a small typo in Lib/test/test__locale.py.
Should read `thousands` rather than `thousauds`.

https://bugs.python.org/issue39009

Automerge-Triggered-By: @pablogsal

### CODE REVIEW

File: Misc/NEWS.d/next/Documentation/2019-12-09-10-12-00.bpo-39002.AfgvfO.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix simple typo in Lib/test/test_statistics.py.
```
Comment (pablogsal) [inline Misc/NEWS.d/next/Documentation/2019-12-09-10-12-00.bpo-39002.AfgvfO.rst]: For fixing simple types, we don't need to add `NEWS` entries. Can you delete it? (I have added the `skip news` label).

**Review by pablogsal — CHANGES_REQUESTED** [review]

File: Misc/NEWS.d/next/Documentation/2019-12-09-10-12-00.bpo-39002.AfgvfO.rst (hunk 1 above)
Comment (timgates42) [reply Misc/NEWS.d/next/Documentation/2019-12-09-10-12-00.bpo-39002.AfgvfO.rst]: thx Done

**Review by pablogsal — APPROVED** [review]
LGTM

Thanks @timgates42 for the contribution! :rocket:

**Comment by timgates42** [comment]
I have made the requested changes; please review again

**Pull request merged**